files/data.yml

```diff
@@ -370,22 +147,10 @@ data:
     port: '15003'
 - name: tut.vo.ibergrid.eu
   voms:
-  - DN: /C=PT/O=LIPCA/O=LIP/OU=Lisboa/CN=voms01.ncg.ingrid.pt
-    CA_DN: /C=PT/O=LIPCA/CN=LIP Certification Authority
+  - DN: /C=PT/ST=Lisboa/Ass.Par.A i"/O="LIP-Lab De Instrumentacao e Fis Expe de Particulas/CN=voms01.ncg.ingrid.pt
```

Comment:
weird quotation here. This is what I see in the browser:
CN = voms01.ncg.ingrid.pt
O = LIP-Lab De Instrumentacao e Fis Expe de Particulas, Ass.Par.A i
ST = Lisboa
C = PT

Reply:
So the organisation field with a coma is apparently creating issues in the parsing by the ops portal.
Will fix it manually.
I'm not really feeling optimistic about the future of this repo, or at least to the current approach for getting the info about the VOs. We still need to have this working till we have fully moved to tokens...

Reply:
It should be better for this DN now